### PR TITLE
Testing - Surge deploy

### PR DESCRIPTION
**Pull request by Shekharrajak**
Should get deployed at http://pr-58-numpy.org-newsite.surge.sh/

### CODE REVIEW

**Comment by Shekharrajak** [comment]
There is 2 scripts : build.sh to generate the build and deploy.sh to deploy it in surge but after the build command inside docker container next commad is not recognizing the` ./public` folder.

**Comment by Shekharrajak** [comment]
When I build the newsite branch then I found that it is building following files  

```
css         images      index.xml   js          sitemap.xml

```

That should be html file instead. @joelachance Please let me know if you already know about it.

**Comment by joelachance** [comment]
@Shekharrajak, I don't know about the index.xml, but I'll take a look at it tomorrow.

**Comment by joelachance** [comment]
@Shekharrajak, check this out:

https://stackoverflow.com/questions/42470517/hugo-not-generating-index-and-404-html-files

According that post, it seems we may need to build the submodules. I'm thinking we may want to try `git submodule update --init` (the command in the README) in line 2 of `build.sh`.

File: .scripts/build.sh [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
```
Comment (joelachance) [inline .scripts/build.sh]: Should we delete this file since its not being used, or do we still plan to move the build steps here?

**Comment by joelachance** [comment]
Should be working now! Looks good to merge.

File: .scripts/build.sh (hunk 1 above)
Comment (Shekharrajak) [reply .scripts/build.sh]: I will make a note. Actually previously in this PR: https://github.com/numpy/numpy.org/pull/46 , I used to call the `build.sh` and `deploy.sh` separately. But now we were facing [issue](https://github.com/numpy/numpy.org/pull/58#issuecomment-544115089) so I make it in one script file.

**Comment by Shekharrajak** [comment]
Thanks @joelachance

**Comment by Shekharrajak** [comment]
I am merging it if no one has any comments on this PR by tomorrow.

**Comment by rgommers** [comment]
LGTM, merging. Thanks @Shekharrajak!

File: .scripts/build.sh (hunk 1 above)
Comment (joelachance) [reply .scripts/build.sh]: Got it! I did see your comment in deploy.sh after I made this comment. Maybe we just delete or add a `#deprecated / not being used` comment in build.sh?

**Pull request merged**